testng.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name = "Suite1">
```

Comment:
create another one for difference, with a list of test methods

<test name="">
        <classes>
            <class name="">
                <methods>
                    <include name=""/>
                    <include name=""/>
                </methods>
            </class>
        </classes>
    </test>

and add to any of the suites such config

<suite name="" parallel="tests" thread-count="1">